apps/web/components/walkthrough/Walkthrough.tsx

```diff
@@ -37,12 +39,25 @@ export default function Walkthrough({
     // setVariable(currentQuestion.variableToSet);
```

Comment:
Do we want to remove this commented code?

Reply:
Good call. Removed!